search/src/Schemas/AgContext.json

```diff
@@ -27,31 +27,46 @@
       "description": "A number identifying the AgContext."
     },
     "crop_type": {
-      "type": "string",
-      "enum": [
-        "wheat",
-        "barley",
-        "oats",
-        "triticale",
-        "sorghum",
-        "rice",
-        "lupins",
-        "chickpeas",
-        "lentils",
-        "field_peas",
-        "faba_beans",
-        "mung_beans",
-        "canola",
-        "mustard",
-        "flax",
-        "linseed",
-        "safflower",
-        "cotton",
-        "carrots",
-        "weed_only",
-        "other"
-      ],
-      "description": "Crop type.\nOne of several strings describing the crop grown in the image."
+      "title": "Crop Type",
+      "oneOf": [
+        {
+          "type": "string",
+          "title": "Grain Crop",
+          "enum": [
+            "wheat",
+            "barley",
+            "oats",
+            "triticale",
+            "sorghum",
+            "rice",
+            "lupins",
+            "chickpeas",
+            "lentils",
+            "field_peas",
+            "faba_beans",
+            "mung_beans",
+            "canola",
+            "mustard",
+            "flax",
+            "linseed",
+            "safflower",
+            "cotton"
```

Comment:
Cotton isn't technically a grain crop, more of a fibre crop but it's been there from the beginning so I think just leave it there for now.